src/testlib.ts

```diff
@@ -1,29 +1,82 @@
 'use-strict';
 import * as Ajv from 'ajv';
+import * as path from 'path';
+import * as fs from 'fs';
 
 const ajv = new Ajv({
   allErrors: true,
 });
 
-const SYNC_RESPONSE_SCHEMA = require('../intents/sync.response.schema.json');
-const QUERY_RESPONSE_SCHEMA = require('../intents/query.response.schema.json');
-const EXECUTE_RESPONSE_SCHEMA = require('../intents/execute.response.schema.json');
-const DISCONNECT_RESPONSE_SCHEMA = require('../intents/disconnect.response.schema.json');
-const ON_OFF_STATES_SCHEMA = require('../traits/onoff.states.schema.json');
-const ON_OFF_ATTRIBUTES_SCHEMA = require('../traits/onoff.attributes.schema.json');
-
+/**
+ * Retrieve the JSON schema for the given intent.
+ * @param intentName FQN of the intent
+ * @param schemaType One of 'request' or 'response'
+ */
+function getIntentSchema(intentName: string, schemaType: string): object | undefined {
```

Comment:
nit: what about merging those into a `requireSchema` function, w/ a check/indirection on the type to compute the path?

Reply:
Done